detection_rules/beats.py

```diff
@@ -72,8 +72,11 @@ def _flatten_schema(schema: list, prefix="") -> list:
             flattened.extend(_flatten_schema(s["fields"], prefix=prefix + s["name"] + "."))
         elif "fields" in s:
             flattened.extend(_flatten_schema(s["fields"], prefix=prefix))
-        elif "type" in s:
+        elif "name" in s and "description" in s:
```

Comment:
now what happens if none of these are satisfied, but `type` is in `s`?

Reply:
@rw-access

Reply:
we could remove description. but `name` is required. see L106 or L129